internal/auth/auth.go

```diff
@@ -36,6 +36,7 @@ var requiredScopes = []string{
 	"read:connections", "update:connections",
 	"read:client_keys", "read:logs", "read:tenant_settings", "read:custom_domains",
 	"read:anomaly_blocks", "delete:anomaly_blocks",
+	"create:log_streams", "delete:log_streams", "read:log_streams", "update:log_streams",
```

Comment:
There's a small test for these in `auth_test.go`. Do you mind adding `log_streams` to the list of CRUD resources? (https://github.com/auth0/auth0-cli/blob/main/internal/auth/auth_test.go#L7-L12)